configure.ac

```diff
@@ -1346,6 +1359,11 @@ PACEMAKER_CONFIG_DIR="${sysconfdir}/pacemaker"
 AC_DEFINE_UNQUOTED(PACEMAKER_CONFIG_DIR,"$PACEMAKER_CONFIG_DIR", Where to keep configuration files like authkey)
 AC_SUBST(PACEMAKER_CONFIG_DIR)
 
+if test x"${localedir}" = x""; then
+    localedir="${datadir}/locale"
+fi
```

Comment:
The 3 lines above are the default handling that is now done by expand_path, so these can be dropped